docs/handshake.md

```diff
@@ -38,100 +67,212 @@ rules. Otherwise, if both sides are at SRT version 1.3.0 or greater, *HSv5* is
+   |0|                     Packet Sequence Number                  |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |FF |O|KK |R|                  Message Number                   |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                          Time Stamp                           |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                    Destination Socket ID                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+**NOTE:** Packet diagrams in this document are in network bit order.
+
+While a complete description of a data packet is out of scope for this document, 
+here is a description of some other header fields unique to SRT:
+
+- **FF** = (2 bits) Position of packet in message, where:
+  - 10b = 1st
+  - 00b = middle
+  - 01b = last
+  - 11b = single
+
+- **O** = (1 bit) Indicates whether the message should be delivered in order (1) 
+or not (0). In File/Message mode (original UDT with UDT_DGRAM) when this bit is 
+clear then a message that is sent later (but reassembled before an earlier message 
+which may be incomplete due to packet loss) is allowed to be delivered immediately, 
+without waiting for the earlier message to be completed. This is not used in Live 
+mode because there's a completely different function used for data extraction 
+when TSBPD mode is on.
+
+- **KK** = (2 bits) Indicates whether or not data is encrypted:
+  - 00b: not encrypted
+  - 01b: encrypted with even key 
+  - 10b: encrypted with odd key
+
+- **R** = (1 bit) Retransmitted packet. This flag is clear (0) when a packet is 
+transmitted the very first time, and is set (1) if the packet is retransmitted.
+
+In **Data** packets, the third and fourth fields are interpreted as follows:
+
+- `PH_TIMESTAMP`: Usually the time when a packet was sent, although the real
 interpretation may vary depending on the type, and it's not important for the
 handshake
-
-Most important here are first two fields, the interpretation of which mainly
-depends on the most significant bit in the `PH_SEQNO` major field:
-
- - 1: Control packet
- - 0: Data (payload) packet
-
-Details for Data packet will be discussed at **Extension flags**. For Control packet
-these two fields are interpreted respectively as:
+- `PH_ID`: The **Destination Socket ID** to which a packet should be dispatched, 
+ although it may have the special value 0 when the packet is a connection request
+
+Additional details for Data packets will be discussed in the sections below 
+covering **extension flags**. 
+
+An SRT control packet header ("packet type" bit = 1) has the following structure:
+
+```
+   0                   1                   2                   3
+   0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |1|          Message Type         |    Message Extended Type    |
```

Comment:
I believe the bar should be directly under the "6", not the "7".

Reply:
Yes, and the less significant digit bar should be also better aligned.